.github/workflows/run_create_lock.yml

```diff
@@ -0,0 +1,37 @@
+# This is a  workflow that will run the unit tests and upload to Codecov
+
+name: create lock
```

Comment:
we should not be creating lock files on the GitHub

Reply:
just for test lock mechanism